content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+---
+title: June '21 Community Gems
+date: 2021-06-25
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-05-25/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/june-21-community-gems/779
+tags:
+  - Community
+  - Pipelines
+  - Remote Storage
+---
+
+### [Q: Is it possible to plot multiple experiments together?](https://discord.com/channels/485586884165107732/563406153334128681/834387923482181653)
```

Comment:
What is the rule - I see that in the previous gems we were not using this `Q:` prefix?

Reply:
It looks like we were using the `Q:` prefix like this before in the other community gems files, so that's why I went with it.